apps/src/sites/studio/pages/teacher_dashboard/show.js

```diff
@@ -130,8 +130,17 @@ $(document).ready(function () {
   };
 
   const getV2TeacherDashboard = () => {
-    const selectedSectionFromList = sections.find(s => s.id === section.id);
-    const selectedSection = {...selectedSectionFromList, ...section};
+    let selectedSection;
+    let selectedSectionFromList;
```

Comment:
optional nit: we could do

`const selectedSectionFromList = window.location.pathname === '/teacher_dashboard/home' ? sections[0] : sections.find(s => s.id === section.id);
const selected section = {...selectedSectionFromList, ...section}`